python/cudf/cudf/_fuzz_testing/utils.py

```diff
@@ -192,11 +222,79 @@ def pandas_to_avro(df, file_name=None, file_io_obj=None):
         fastavro.writer(file_io_obj, avro_schema, records)
 
 
+def _preprocess_to_orc_tuple(df):
+    def _null_to_None(value):
+        if value is pd.NA or value is pd.NaT:
+            return None
+        else:
+            return value
+
+    has_nulls_or_nullable_dtype = any(
+        [
+            True
+            if df[col].dtype in pandas_dtypes_to_cudf_dtypes
+            or df[col].isnull().any()
+            else False
+            for col in df.columns
+        ]
+    )
+
+    tuple_list = []
+    for tup in df.itertuples(index=False, name=None):
+        if has_nulls_or_nullable_dtype:
+            tuple_list.append(tuple(map(_null_to_None, tup)))
+        else:
+            tuple_list.append(tup)
+
+    return tuple_list
+
+
+def pandas_to_orc(df, file_name=None, file_io_obj=None, stripe_size=67108864):
+    schema = get_orc_schema(df)
+
+    tuple_list = _preprocess_to_orc_tuple(df)
+
+    if file_name is not None:
+        with open(file_name, "wb") as data:
+            with pyorc.Writer(
+                data, str(schema), stripe_size=stripe_size
+            ) as writer:
+                writer.writerows(tuple_list)
+    elif file_io_obj is not None:
+        with pyorc.Writer(
+            file_io_obj, str(schema), stripe_size=stripe_size
+        ) as writer:
+            writer.writerows(tuple_list)
+
+
+def orc_to_pandas(file_name=None, file_io_obj=None, stripes=None):
+    if file_name is not None:
+        f = open(file_name, "rb")
+    elif file_io_obj is not None:
+        f = file_io_obj
+
+    reader = pyorc.Reader(f)
+
+    if stripes is None:
+        df = pd.DataFrame.from_records(
+            reader, columns=reader.schema.fields.keys()
+        )
+    else:
+        records = []
+        for i in stripes:
+            records.extend(list(reader.read_stripe(i)))
```

Comment:
Can use a stacked list comprehension
```suggestion
        records = [record for i in stripes for record in list(reader.read_stripe(i))]
```